ecommerce/migrations/0007_unitofmeasure_productprice.py

```
@@ -0,0 +1,50 @@
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ecommerce', '0006_cart'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UnitOfMeasure',
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)